deliverables/deliverable_2.md

```diff
@@ -0,0 +1,237 @@
+- The system should respond to errors with an error page.
+- The error page should reflect what went wrong with the application.
+- The error page should not be jarring to the user experience.
+- If the error is on the user’s end, provide simple instructions on how to remedy the issue.
+
+
+## 4. Non-functional requirements
+
+**Function Stability** - The App/Website should be running 100% of the time, hyperlinks and buttons should constantly be working once the website is launched.
+
+**Performance Efficiency** - The App/Website should run smoothly. 8/10 users will enjoy the website’s smoothness and ability to be upheld.
+
+**Compatibility** - The App/Website will be cross compatible with all devices. 100% of users will be able to access from all devices that can search for websites.
+
+**Usability** - The App/Website will be easily accessible and pleasing to the user. 8/10 users will be happy with the setup of the site.
+
+**Reliability** - The website will be frequently updated to ensure it’s always reliable. The website should be up 100% of the time once launched.
+
+**Security** - The website will have a login entry to ensure security. 9/10 users will never have security issues.
+
+**Maintainability** - The website will be easy to access, navigate, and will be updated frequently to keep up to date on the best software available. 8/10 users will be able to navigate our website with ease.
+
+**Portability** - App/Website should be able to be used on all devices including mobile. 100% of users will be able to access from all devices.
+
+
+## 5. MVP
+Designing a minimum viable product for a stock tracker app will require only the necessary functionality while still satisfying the requirements for this product. Firstly, an app icon and name will be established.  As for a user interface, the app will have a basic set of features our whole app is based around. The features that will be included are the following: a search bar where investors will search for stocks, the ability to favorite a stock and add it to your list of stocks being tracked, a filter to set at what price jump and/or decrease to be notified at, and a place to view the users favorited stocks and their relative data. These basic features will be the foundation to validating our product. To achieve this we will conduct user studies in which the users will test our MVP for a few days. In these few days they will eventually search for a few desired stocks to track, favorite them, set their preferred notification settings, and then proceed to evaluate whether the stock updates given were helpful with monitoring their favorite stocks by performing a survey after their usage. The feedback for our MVP will allow us to get a better understanding of the value of our product and benefit us in creating the most optimal stock tracker app. 
+
+
+## 6. Use Cases
+
+### 6.1. Use Case Diagrams
+**![Stock Infinity Use Case Diagram](./deliverable_images/Stock_Infinity_Use_Case_Diagram.png)**
```

Comment:
the names of the use cases  are not consistent with the one used in the description